# NGS Capability Assessment for Influenza Surveillance

This questionnaire assesses your laboratory's capabilities for next-generation sequencing (NGS) of clinical influenza specimens, including sample processing, quality control, sequence analysis, and data sharing. Your responses will help us tailor training and support to your specific needs.

---

## Your information

### 1. Name

### 2. Title

### 3. Email

### 4. Institution

### 5. Country

## Laboratory Staff & Experience

### 6. Laboratory's experience with influenza surveillance (select one)

- [ ] New to influenza work (< 6 months)
- [ ] 6 months - 2 years
- [ ] 2-5 years
- [ ] 5+ years

### 7. Please briefly describe your influenza-related work

### 8. Public Health Laboratory Leadership -- List the years of experience each staff member has with performing leadership/management-related tasks (e.g., program oversight, reporting, policy development, etc.)

### 9. IT / Data Management -- List the years of experience each staff member has with performing IT/data management-related tasks (e.g., database management, server administration, data security, etc.)

### 10. Epidemiology -- List the years of experience each staff member has with performing epidemiology-related tasks (e.g., outbreak investigation, surveillance data analysis, etc.)

### 11. Virology -- List the years of experience each staff member has with performing virology-related tasks (e.g., virus isolation, culture, antigenic assays, etc.) 

### 12. Molecular Biology -- List the years of experience each staff member has with performing molecular biology-related tasks (e.g., nucleic acid extraction, PCR, qPCR, library preparation, NGS, etc.)

### 13. Bioinformatics / Computational Biology -- List the years of experience each staff member has with performing bioinformatics-related tasks (e.g., command line, scripting, sequence analysis, phylogenetics, data submission, etc.)

### 14. Laboratory's bioinformatics experience level (select one)

- [ ] No prior experience
- [ ] Basic (some command line, can follow tutorials)
- [ ] Intermediate (comfortable with scripting, pipeline execution)
- [ ] Advanced (develop custom workflows, troubleshoot complex issues)

---

## Laboratory Infrastructure

### 15. Monthly specimen volume

**Approximate number of influenza-positive clinical specimens per month:** 

**Peak season volume:** 

### 16. How would you describe the electrical power stability in your laboratory? (select one)

- [ ] Stable (rare or no outages)
- [ ] Occasional outages (1-2 per month)
- [ ] Frequent outages (weekly or more)
- [ ] Unpredictable / varies by season

### 17. Do you have uninterruptible power supply (UPS) or backup generator for critical equipment? (select all that apply)

- [ ] Yes, UPS for all critical equipment (sequencers, computers, freezers)
- [ ] Yes, UPS for some equipment 
- [ ] Yes, backup generator available
- [ ] No backup power system
- [ ] Planning to install

### 18. What is your laboratory's typical internet _download_ speed? (select one)

- [ ] < 10 Mbps
- [ ] 10-50 Mbps
- [ ] 50-100 Mbps
- [ ] 100-500 Mbps
- [ ] > 500 Mbps
- [ ] Not sure

### 19. What is your laboratory's typical internet _upload_ speed? (select one)

- [ ] < 5 Mbps
- [ ] 5-25 Mbps
- [ ] 25-50 Mbps
- [ ] 50-100 Mbps
- [ ] > 100 Mbps
- [ ] Not sure

### 20. Do you experience internet connectivity issues that impact your work? (select one)

- [ ] No, connection is reliable
- [ ] Occasionally (less than monthly)
- [ ] Frequently (weekly or more)
- [ ] Yes, significant barrier to data transfer/analysis

---

## Next-Generation Sequencing Platform

### 21. Current NGS platform(s) (select all that apply)

- [ ] Illumina MiSeq
- [ ] Illumina NextSeq (specify model: 500/550/1000/2000) 
- [ ] Illumina iSeq 100
- [ ] Illumina NovaSeq
- [ ] Oxford Nanopore MinION
- [ ] Oxford Nanopore GridION
- [ ] Ion Torrent
- [ ] PacBio Sequel
- [ ] Other: 

### 22. How often do you perform NGS runs? (select one)

- [ ] Daily
- [ ] 2-3 times per week
- [ ] Weekly
- [ ] Monthly
- [ ] Less than monthly

### 23. What is the typical number of influenza samples per run?

---

## Sample Processing & Library Preparation

### 24. RNA extraction method (select one)

- [ ] Automated (specify platform) 
- [ ] Manual (specify kit) 
- [ ] Both automated and manual

### 25. RNA quantification method(s) (select all that apply)

- [ ] Qubit Fluorometer (RNA HS/BR assay)
- [ ] TapeStation (specify Agilent model) 
- [ ] Bioanalyzer (specify Agilent model) 
- [ ] NanoDrop spectrophotometer
- [ ] qRT-PCR (Ct value as proxy)
- [ ] Other: 

### 26. Reverse transcription method

- [ ] SuperScript III
- [ ] SuperScript III - HiFi
- [ ] SuperScript IV
- [ ] SuperScript IV - HiFi
- [ ] Other

### 27. Influenza genome amplification strategy (select all that apply)

- [ ] Shotgun Whole genome amplification (random priming/transposase)
- [ ] Targeted amplicon-based
- [ ] Hybrid capture/enrichment
- [ ] Not yet decided

### 28. If using targeted amplicons, which primer scheme do you use for influenza genome amplification? (select all that apply)

- [ ] CDC primers
- [ ] ARTIC primers
- [ ] Custom in-house primers
- [ ] Commercial kit
- [ ] Other

### 29. Library preparation workflow (select all that apply)

- [ ] Nextera XT
- [ ] Nextera Flex
- [ ] TruSeq DNA/RNA
- [ ] NEBNext Ultra II
- [ ] KAPA HyperPrep
- [ ] Swift Normalase / SNAP
- [ ] Oxford Nanopore native barcoding
- [ ] Oxford Nanopore rapid barcoding
- [ ] Other

### 30. Library QC methods (select all that apply)

- [ ] Qubit (dsDNA HS assay)
- [ ] TapeStation / Bioanalyzer (fragment size distribution)
- [ ] qPCR quantification
- [ ] Visual inspection (agarose gel)
- [ ] No formal QC
- [ ] Other

### 31. Library pooling and normalization

- [ ] Manual calculation (Qubit + fragment size)
- [ ] Automated normalization (bead-based)
- [ ] qPCR-based equimolar pooling
- [ ] Equal volume pooling
- [ ] Other

---

## Quality Control

### 32. Do you record RT-PCR Ct values for specimens submitted for sequencing? (select one)

- [ ] Yes, systematically
- [ ] Sometimes
- [ ] No

### 33. What is your Ct threshold for attempting sequencing?

### 34. Describe the QC metrics you monitor for sequencing runs.

### 35. Describe your read-level QC.

### 36. Describe your assembly-level QC.

### 37. Describe your consensus-level QC.

---

## Bioinformatics Infrastructure

### 36. What computing resources are available to your lab? (select all that apply)

- [ ] Standard desktop/laptop (Windows)
- [ ] Standard desktop/laptop (Mac)
- [ ] Linux workstation
- [ ] High-performance computing (HPC) cluster
- [ ] Cloud computing (AWS, Azure, Google Cloud)
- [ ] Institutional shared server
- [ ] None currently available

### 37. What operating systems are your lab familiar with? (select all that apply)

- [ ] Windows
- [ ] macOS
- [ ] Linux (Ubuntu, CentOS, etc.)
- [ ] Windows Subsystem for Linux (WSL)

### 38. How proficient is your laboratory on the command-line (computer terminal)? (select one)

- [ ] No experience
- [ ] Can follow written instructions / tutorials
- [ ] Can write simple bash/shell scripts
- [ ] Proficient with scripting (bash, Python, R) and environment management (conda, virtualenv)

### 39. Programming/scripting languages used (select all that apply)

- [ ] None
- [ ] Bash/Shell scripting
- [ ] Python
- [ ] R
- [ ] Perl
- [ ] Java
- [ ] Other: 

### 40. Version control / reproducibility tools (select all that apply)

- [ ] Git / GitHub
- [ ] Docker containers
- [ ] Conda / Bioconda environments
- [ ] Singularity / Apptainer
- [ ] Nextflow / Snakemake workflows
- [ ] None currently

---

## Sequence Assembly & Analysis

### 41. Current bioinformatics tools for influenza NGS (select all that apply)

**Read preprocessing:**
- [ ] IRMA (CDC)
- [ ] MIRA (CDC)
- [ ] Trimmomatic
- [ ] fastp
- [ ] Cutadapt
- [ ] BBDuk / BBTools
- [ ] TrimGalore
- [ ] Other: 

**Read alignment/mapping:**
- [ ] IRMA (CDC)
- [ ] MIRA (CDC)
- [ ] BWA (mem, aln)
- [ ] Bowtie2
- [ ] Minimap2
- [ ] STAR
- [ ] Other: 

**Consensus calling:**
- [ ] IRMA (CDC)
- [ ] MIRA (CDC)
- [ ] SAMtools / BCFtools
- [ ] GATK
- [ ] iVar
- [ ] Custom scripts
- [ ] Other: 

**Minor variant calling (intrahost):**
- [ ] IRMA (CDC)
- [ ] MIRA (CDC)
- [ ] LoFreq
- [ ] VarScan2
- [ ] iVar variants
- [ ] GATK HaplotypeCaller
- [ ] Not currently performing
- [ ] Other: 

**De novo assembly (if applicable):**
- [ ] SPAdes
- [ ] MEGAHIT
- [ ] Trinity
- [ ] Flye (ONT long reads)
- [ ] Not applicable
- [ ] Other: 

### 42. Integrated / commercial analysis platforms currently used

- [ ] BaseSpace (Illumina)
- [ ] CLC Genomics Workbench
- [ ] Geneious (Prime / Biologics)
- [ ] Galaxy platform
- [ ] MIRA (CDC)
    - [ ] Graphical interface
    - [ ] Command-line
- [ ] Terra: <What workflow:version?>
- [ ] None
- [ ] Other: 

### 43. Workflow automation

**Do you use automated/reproducible workflows?**

- [ ] Yes, using Nextflow
- [ ] Yes, using Snakemake
- [ ] Yes, using WDL / Cromwell
- [ ] Yes, using Galaxy
- [ ] Yes, custom scripts
- [ ] No, manual step-by-step
- [ ] Transitioning to automated workflows

### 44. Reference genomes used

**Where do you obtain influenza reference sequences?**

- [ ] NCBI
- [ ] GISAID
- [ ] Internal lab database
- [ ] Manually curated set
- [ ] Automatic selection by software
- [ ] Not sure

---

## Phylogenetic & Clade Assignment

### 45. Clade/subclade assignment tools (select all that apply)

- [ ] Nextclade (web or CLI)
- [ ] OCTOFLU 
- [ ] MIRA (CDC)
- [ ] GISAID - FluSurver
- [ ] Manual BLAST comparison
- [ ] Custom phylogenetic placement
- [ ] Not currently performed
- [ ] Other: 

### 46. Multiple sequence alignment tools

- [ ] MAFFT
- [ ] Muscle
- [ ] Clustal Omega
- [ ] Built into analysis software (Geneious, MEGA, etc.)
- [ ] Other: 

### 47. Phylogenetic tree construction methods used (select all that apply)

- [ ] Neighbor-Joining (NJ)
- [ ] Maximum Likelihood (ML)
- [ ] Maximum Parsimony
- [ ] Bayesian (BEAST, MrBayes)
- [ ] Unknown / built into software
- [ ] Not currently performed
- [ ] Other: 

### 48. Phylogenetic software currently used (select all that apply)

- [ ] Nextstrain / Augur
- [ ] Nextclade
- [ ] IQ-TREE
- [ ] RAxML / RAxML-NG
- [ ] FastTree
- [ ] MEGA (X or 11)
- [ ] PhyML
- [ ] BEAST / BEAST2
- [ ] MrBayes
- [ ] Geneious tree builder
- [ ] None
- [ ] Other: 

### 49. Tree visualization and annotation

- [ ] FigTree
- [ ] ggtree (R package)
- [ ] Microreact
- [ ] Nextstrain / Auspice
- [ ] Nextclade
- [ ] Geneious
- [ ] MEGA
- [ ] Other: 

---

## Genetic Marker & Mutation Detection

### 50. Genetic marker and mutation analysis performed (select all that apply)

**Do you routinely screen for the following genetic markers and mutations?**

- [ ] Antiviral resistance mutations
- [ ] HA cleavage site analysis
- [ ] Glycosylation site prediction
- [ ] Epitope/antigenic site mutation tracking
- [ ] None currently performed

**Analysis approach:**
- [ ] Automated in pipeline
- [ ] Manual inspection of alignments
- [ ] On request / case-by-case basis
- [ ] Not applicable

**Tools/databases used for mutation detection (select all that apply):**

- [ ] MIRA (CDC)
- [ ] GISAID FluSurver
- [ ] Nextclade
- [ ] NetNGlyc
- [ ] Custom scripts with curated mutation list
- [ ] Manual review of alignments
- [ ] Other: 


---

## Data Management & Sharing

### 51. Sequence data storage

**Where do you store raw sequencing reads (FASTQ files)?**

- [ ] Local server / NAS
- [ ] Cloud storage (AWS S3, Azure Blob, Google Cloud)
- [ ] External hard drives
- [ ] Institutional HPC storage
- [ ] Vendor platform (BaseSpace, etc.)
- [ ] Not yet storing systematically

**Approximate current storage capacity:** 

### 52. Sequence metadata management

**How do you track sample metadata (collection date, location, patient demographics, etc.)?**

- [ ] Excel / CSV spreadsheets
- [ ] Laboratory Information Management System (LIMS)
- [ ] REDCap or similar database
- [ ] Custom database (SQL, Access, etc.)
- [ ] Paper records only
- [ ] Other: 

### 53. Public database submission experience

**Which databases have you submitted influenza sequences to? (select all that apply)**

- [ ] NCBI GenBank / SRA
- [ ] GISAID EpiFlu
- [ ] NCBI BioProject / BioSample
- [ ] European Nucleotide Archive (ENA)
- [ ] None yet
- [ ] Other: 

### 54. Data submission tools/workflows used

- [ ] NCBI Submission Portal (web)
- [ ] NCBI FTP upload
- [ ] GISAID Submission tool (web)
- [ ] GISAID CLI tool
- [ ] ENA Submission Portal (web)
- [ ] ENA Webin (CLI)
- [ ] SeqSender (CDC tool)
- [ ] Other command-line tools (e.g., NCBI SRA toolkit)
- [ ] Not yet submitting
- [ ] Other: 

### 55. Data sharing policies

**Are you able to share sequence data publicly in real-time?**

- [ ] Yes, immediately upon QC approval
- [ ] Yes, after internal review (delay: _______)
- [ ] Yes, after publication or approval from collaborators
- [ ] Restrictions due to policy/ethics
- [ ] Not sure

### 56. Data governance & privacy

**Do you have institutional policies for genomic data sharing and patient privacy?**

- [ ] Yes, well-defined policies in place
- [ ] Policies in development
- [ ] No formal policies
- [ ] Not sure

---

## Quality Management & Documentation

### 57. Standard Operating Procedures (SOPs)

**Do you have written SOPs for:**

- [ ] RNA extraction from clinical specimens
- [ ] RT-PCR and library preparation
- [ ] NGS run setup and QC
- [ ] Bioinformatics analysis pipeline
- [ ] Data submission and archiving
- [ ] None currently documented

### 58. Quality assurance practices

- [ ] Positive and negative controls in each run
- [ ] Routine participation in proficiency testing / EQA programs
- [ ] Internal replicate testing
- [ ] Regular equipment calibration and maintenance logs
- [ ] Documented incident reporting and corrective actions
- [ ] Not yet implemented

### 59. Laboratory accreditation

- [ ] ISO 15189 (Medical laboratories)
- [ ] ISO 17025 (Testing laboratories)
- [ ] CAP accredited
- [ ] CLIA certified
- [ ] Other: 
- [ ] Not accredited
- [ ] Accreditation in progress

---

## Training & Support Needs

### 60. Areas where you need additional training or support (select all that apply)

**Wet lab / molecular:**
- [ ] Advanced library preparation techniques
- [ ] Troubleshooting low-input / degraded samples
- [ ] Optimizing amplicon schemes
- [ ] NGS QC and troubleshooting

**Bioinformatics:**
- [ ] Command-line / Linux basics
- [ ] Read QC and preprocessing
- [ ] Genome assembly and consensus calling
- [ ] Variant calling and interpretation
- [ ] Phylogenetic analysis
- [ ] Pipeline automation (Nextflow, Snakemake)
- [ ] Cloud computing setup

**Data management:**
- [ ] Metadata standards and best practices
- [ ] Database submission workflows
- [ ] Setting up LIMS or data management systems
- [ ] Data sharing policies and compliance

**Interpretation:**
- [ ] Clade/subclade assignment
- [ ] Identifying antiviral resistance mutations
- [ ] Outbreak investigation and molecular epidemiology
- [ ] Integration with phenotypic assay results

### 61. Preferred training format

- [ ] In-person hands-on workshop
- [ ] Virtual instructor-led sessions
- [ ] Self-paced online modules
- [ ] One-on-one mentoring / consultation
- [ ] Written protocols and documentation

### 62. Primary goals for NGS capability (select top 3)

- [ ] Real-time surveillance and outbreak detection
- [ ] Clade/strain characterization for vaccine strain selection
- [ ] Antiviral resistance monitoring
- [ ] Zoonotic spillover detection (avian, swine influenza)
- [ ] Co-infection and reassortment detection
- [ ] Within-host diversity / transmission studies
- [ ] Data sharing with WHO / public health networks
- [ ] Research and publications
- [ ] Other: 

### 63. Current challenges or obstacles (select all that apply)

- [ ] Limited funding for reagents/consumables
- [ ] Lack of trained personnel
- [ ] Insufficient computational resources
- [ ] Inconsistent specimen quality
- [ ] Low influenza positivity rates
- [ ] Difficulty accessing reference materials or controls
- [ ] Data storage and management capacity
- [ ] Internet bandwidth for large file transfers
- [ ] Regulatory / ethics approval processes
- [ ] Integration with existing laboratory workflows
- [ ] Other: 

### 64. Additional comments or specific questions

Please share any additional information about your laboratory's capabilities, challenges, or specific areas where you would like support:



---

**Thank you for completing this assessment. Your input will help us provide targeted training and technical support for your influenza NGS program.**
